README.md

```diff
@@ -1,42 +1,126 @@
+
+### Step 4: Standardizing and deploying your plugin
+
+If you're satisfied with the tests performed and the developed plugin:
+
+1. Access your profile on [npm](https://www.npmjs.com/). As a part of the `vtex` organization, you'll have access to [generate a new **publish** token.](https://docs.npmjs.com/creating-and-viewing-access-tokens)
+2. Save the generated token.
+3. Open your plugin repository on GitHub.
+4. Go to *Settings > Secrets* and [create a new repository secret](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) with the name `NPM_TOKEN` and paste the value of the previously generated token.
+
+>⚠️ *Please use capital letters for `NPM_TOKEN`.*
+
+5. Now, go to *Settings > Manage access* and include the [IO DevTools](https://github.com/orgs/vtex/teams/io-devtools) and [IO Framework](https://github.com/orgs/vtex/teams/io-framework) teams as admins.
+
+>⚠️ *If you're already part of one of those teams, please consider removing your single access.*
+
+6. In the *Settings > Branches* section, [add a new rule](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuring-protected-branches) to the master branch, making sure you check the following options:
+
+- Require pull request reviews before merging (Required approving reviews: 1).
+- Require status checks to pass before merging.
+    - Require branches to be up to date before merging.
+
+7. Open a pull request containing the changes in your plugin repository. 
+8. Wait for comments from reviewers, discuss and fix any related issues. Once you receive an LGTM, deploy your plugin using the [releasy](https://github.com/vtex/releasy#usage) package.
+
+Once the CI tests are completed, your package will be automatically available on [`npm`](https://www.npmjs.com/) as part of the `vtex` organization.
+
+### [OPTIONAL] Step 5: Adding your new plugin to the Toolbelt
+
+The [Toolbelt](https://github.com/vtex/toolbelt) project adheres to a strict pull request review workflow.
+
+To request including your plugin in the Toolbelt, update the [Toolbelt `dependencies`](https://github.com/vtex/toolbelt/blob/master/package.json#L52) and the [`CHANGELOG.md`](https://github.com/vtex/toolbelt/blob/master/CHANGELOG.md) file. Open a pull request, explaining what your plugin does, wait for comments from reviewers, discuss and fix any related issues. Once you receive an LGTM, you'll be able to merge it to the master branch.
+
+>⚠️ ***Keep in mind:** If your plugin is not accepted in the [Toolbelt](https://github.com/vtex/toolbelt) project, you can still use it **locally**. For that, you must run `yarn add {path}` in the **cloned** Toolbelt folder, where `{path}` must be replaced with the [npm](https://www.npmjs.com/) or relative path of your plugin. In these cases, however, VTEX does not support, neither takes any responsibility for any damages caused by the developed plugin.*
```

Comment:
I think this part was no clear and correct enough.

1 - To run a custom plugin locally you need to go inside de cloned `toolbelt` repo and fire the command `yarn add PLUGIN_NAME`, then run `yarn watch` and use `vtex-test` to use it.

2 - Fortunately the new `toolbelt` major (3.x) will contain a new command called `plugins` that will enable a user to install plugins directly inside `toolbelt`, for exemple: `vtex plugins install lighthouse` that will find for `@vtex/cli-plugin-lighthouse` on `npm`. For third parties plugins you can use `vtex plugins install PLUGIN_NAME` that can be in the format `@ORG/PLUGIN_NAME`